ntbs-service/Models/Countries.cs

```diff
@@ -1,14 +1,74 @@
 using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using Microsoft.EntityFrameworkCore.Internal;
 
 namespace ntbs_service.Models
 {
     public static class Countries
     {
         public const string UkCode = "GB";
         public const string UnknownCode = "-";
-        public static Country[] GetCountriesArray()
+
+        public static readonly string[] notHighTbOccurenceCountries = new[]
+        {
+            "Andorra",
+            "Australia",
+            "Austria",
+            "Belgium",
+            "Canada",
+            "Cyprus",
+            "Denmark",
+            "Finland",
+            "France",
+            "Germany",
+            "Greece",
+            "Greenland",
+            "Guernsey",
+            "Holy See (Vatican City State)",
+            "Iceland",
+            "Ireland",
+            "Isle Of Man",
+            "Italy",
+            "Jersey",
+            "Liechtenstein",
+            "Luxembourg",
+            "Monaco",
+            "Netherlands",
+            "New Zealand",
+            "Norway",
+            "Portugal",
+            "Spain",
+            "Sweden",
+            "Switzerland",
+            "United Kingdom",
+            "United States",
+            "United States Minor Outlying Islands"
+        };
+
+        public static IEnumerable<Country> GetCountriesArray()
         {
-            return new Country[] {
+            var allCountries = GetAllCountryData().ToList();
+
+            // Set all countries to high occurence and conditionally flag them as low.
+            // The vast majority are high occurence.
+            allCountries.ForEach(c => c.HasHighTbOccurence = true);
+
+
+            foreach (var notHighTbCountry in notHighTbOccurenceCountries)
+            {
+                // If country not found, or duplicate countries found with the name, this will throw an exception.
+                // Couldn't come up with a neat strongly typed way of doing this.
+                var lowCountry = allCountries.Single(c => c.Name == notHighTbCountry);
+                lowCountry.HasHighTbOccurence = false;
+            }
+
+            return allCountries;
+        }
+
+        private static IEnumerable<Country> GetAllCountryData()
+        {
+            return new[] {

```

Comment:
This approach was used for the initial population of the countries table, I can see how having it accessible on the model class is a bit iffy, just adapted what was there for this work. Have you a preference where this would be?

I could shift the country/HighTb logic to a seedData method?